.github/workflows/up-to-date.yml

```diff
@@ -0,0 +1,25 @@
+name: Banch up-to-date with target
+
+on: pull_request
+
+jobs:
+    compare:
+        name: Confirm that the branch is up-to-date with its target
+
+        # We can't currently run benchmarks on PRs from forked repos, because the
+        # tachometer action reports results by posting a comment, and we can't post
+        # comments without a github token.
+        if: github.event.pull_request == null || github.event.pull_request.head.repo.full_name == github.repository
```

Comment:
Since you're triggering this workflow from the pull_request event, the value of event.pull_request should never be null.  Here's the object you should have access to on this event: https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#pull_request

Reply:
Nice! Lots of copy/pasta in the actions code to date. Anytime we can get rid of things is win 🎊